src/client/components/AchievementComponent/AchieveComponent.stories.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import AchieveComponent from './AchieveComponent';
+import { withKnobs, number } from '@storybook/addon-knobs';
+
+export default { title: 'Achievement Component', decorators: [withKnobs] };
+export const Component = () => (
+  <AchieveComponent
+    achieveTask={number('achieve task', 1)}
+    Ranking={number('Ranking', 3)}
+    timeSpent={number('spent time', 3000)}
```

Comment:
I am not good in math's 😝, i don't want to calculate every time if i want to insert the time here. It will be helpful if you convert it in minutes since it is also showing time in the minutes on the screen 🙂

Reply:
I've converted to minutes, Satisfied? :)

Reply:
Hey @priyankapandey13 I dont understand here why you will put the time in minuts derictly ? the spendTime here it will comes from the counter, so its neccessery to calulatie it here ... but now this time comes from storybook as value for testink the code